lib/elixometer.ex

```diff
@@ -374,7 +374,8 @@ defmodule Elixometer do
       if reporter do
         metric_name
         |> :exometer.info
-        |> Keyword.get(:datapoints)
+        |> Keyword.get(:value)
```

Comment:
Backwards incompatible change in exometer_core. I wonder if this would affect any users?

Reply:
Can this be written in a backwards compatible way, trying `:value`, and then `:datapoints` if `nil` ?

Reply:
Do you mean to keep support of exometer_core 1.4? Yes I'll do that if dependencies can align.